openwisp_radius/settings.py

```diff
@@ -244,3 +244,4 @@ def get_default_password_reset_url(urls):
     ow_users_settings.EXPORT_USERS_COMMAND_CONFIG["select_related"].extend(
         ["registered_user"]
     )
+BATCH_ASYNC_THRESHOLD = get_settings_value("BATCH_ASYNC_THRESHOLD", 15)
```

Comment:
Please add this setting to the settings.rst docs page, I hadn't noticed this in my previous reviews as I focused mostly on testing the functionality and reviewing the code, now I was about to approve but I didn't see any documentation update.

We can add this above `OPENWISP_RADIUS_BATCH_DEFAULT_PASSWORD_LENGTH`.